### PR TITLE
OSSM-4341: Add conceptual images to adding projects

### PR DESCRIPTION
**Pull request by rh-tokeefe**
Version(s): 4.10+

Issue:https://issues.redhat.com/browse/OSSM-4341

Link to docs preview: https://61880--docspreview.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-create-mesh.html#ossm-about-adding-projects-using-smmr_ossm-create-mesh

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/ossm-about-adding-projects-using-smm.adoc [hunk 1]
```diff
@@ -5,6 +5,8 @@
 [id="ossm-about-adding-projects-using-smm_{context}"]
 = About adding projects using the ServiceMeshMember resource
 
-A `ServiceMeshMember` resource provides a way to add a project to a service mesh without modifying the `ServiceMeshMemberRoll` resource. To add a project, create a `ServiceMeshMember` resource in the project that you want to add to the service mesh. When the {SMProductShortName} Operator processes the `ServiceMeshMember` object, the project appears in the `status.members` list of the `ServiceMeshMemberRoll` resource. Then, the services that reside in the project are made available to the mesh. 
+A `ServiceMeshMember` resource provides a way to add a project to a service mesh without modifying the `ServiceMeshMemberRoll` resource. To add a project, create a `ServiceMeshMember` resource in the project that you want to add to the service mesh. When the {SMProductShortName} Operator processes the `ServiceMeshMember` object, the project appears in the `status.members` list of the `ServiceMeshMember` resource. Then, the services that reside in the project are made available to the mesh.
```
Comment (luksa) [inline modules/ossm-about-adding-projects-using-smm.adoc:8]: Please revert this change. The `status.members` is in the `ServiceMeshMemberRoll` resource.

**Review by luksa — APPROVED** [review]
Looks good for dev.

File: modules/ossm-about-adding-projects-using-smm.adoc (hunk 1 above)
Comment (rh-tokeefe) [reply modules/ossm-about-adding-projects-using-smm.adoc]: Thanks for catching this Marko. The change was made and appears in the preview build.

**Review by pbajjuri20 — APPROVED** [review]

File: modules/ossm-about-adding-projects-using-label-selectors.adoc [hunk 2]
```diff
@@ -6,7 +6,11 @@
 [id="ossm-about-adding-projects-using-label-selectors_{context}"]
 = About adding projects using label selectors
 
-For cluster-wide deployments, you can use label selectors to add projects to the mesh. Label selectors specified in the `ServiceMeshMemberRoll` resource enable the {SMProductShortName} operator to add or remove namespaces to or from the mesh based on namespace labels. Unlike other standard {product-title} resources that you can use to specify a single label selector, you can use the `ServiceMeshMemberRoll` resource to specify multiple label selectors. If the labels for a namespace match any of the selectors specified in the `ServiceMeshMemberRoll` resource, then the namespace is included in the mesh.
+For cluster-wide deployments, you can use label selectors to add projects to the mesh. Label selectors specified in the `ServiceMeshMemberRoll` resource enable the {SMProductShortName} operator to add or remove namespaces to or from the mesh based on namespace labels. Unlike other standard {product-title} resources that you can use to specify a single label selector, you can use the `ServiceMeshMemberRoll` resource to specify multiple label selectors.
```
Comment (shipsing) [inline modules/ossm-about-adding-projects-using-label-selectors.adoc:9]:
```suggestion
For cluster-wide deployments, you can use label selectors to add projects to the mesh. Label selectors specified in the `ServiceMeshMemberRoll` resource enable the {SMProductShortName} Operator to add or remove namespaces to or from the mesh based on namespace labels. Unlike other standard {product-title} resources that you can use to specify a single label selector, you can use the `ServiceMeshMemberRoll` resource to specify multiple label selectors. 
```

**Review by shipsing — COMMENTED** [review]
a minor comment. rest of the content looks good.

**Review by shipsing — APPROVED** [review]

File: modules/ossm-about-adding-projects-using-label-selectors.adoc (hunk 2 above)
Comment (rh-tokeefe) [reply modules/ossm-about-adding-projects-using-label-selectors.adoc]: Thanks @shipsing! Change submitted.
